scripts/tests/app/web/material.json

```diff
@@ -5,6 +5,5 @@
   "layout": "simple",
   "auth": false,
   "lazy": false,
-  "angulartics": false,
-  "location": "hash"
```

Comment:
same

Reply:
reverted :) I totally misread some of the generator code